### PR TITLE
bring back browser notifications

### PR DESCRIPTION
**Pull request by patosullivan**
If the user has already given us permission to show them notifications and we have some unreads, send the notification.

### CODE REVIEW

File: ui/src/app.tsx [hunk 1]
```diff
@@ -51,6 +58,23 @@ const AppRoutes = () => {
...
+  useEffect(() => {
+    if (count > 0 && browserNotifications && 'Notification' in window) {
+      unreadNotifications.forEach((bin) => {
+        const rope = bin.topYarn?.rope;
+        // need to capitalize desk name
+        const app = rope?.desk.slice(0, 1).toUpperCase() + rope?.desk.slice(1);
+        const type = getNotificationType(rope);
+        const ship = bin.topYarn?.con.find(isYarnShip)?.ship;
+        new Notification(`Landscape: ${app}`, {
+          body: `${ship}${bin.topYarn.con[1]}${bin.topYarn.con[2]}`,
```
Comment (arthyn) [inline ui/src/app.tsx:73]: does this work for all notifications?

**Review by arthyn — APPROVED** [review]
lgtm, not sure about the body

File: ui/src/app.tsx (hunk 1 above)
Comment (patosullivan) [reply ui/src/app.tsx:73]: I'll play with it and try to break it.

**Comment by patosullivan** [comment]
@arthyn cleaned it up a bit, accounted for the "emph" in the third yarn array item, switched to using the browser's notification preferences, moved the building of the notification out to its own function.

**Review by arthyn — APPROVED** [review]
lgtm

**Pull request merged**